From be8d7dfe7f8cce59588cf2354397efb427343438 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 23 Feb 2024 11:21:26 +0100
Subject: [PATCH 0688/1695] dts: stm32: enable PKA on stm32mp135f-dk

Sets PKA peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1c640fffbdf7ed3b51428f3ba832510f844cc45c dts: stm32: enable PKA on stm32mp135f-dk]
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/282105
Change-Id: If844e14f275ffd52e6f2e0284354824f8db8de93
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362407
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 55b6b221b..458a42f10 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -386,6 +386,10 @@
 	};
 };
 
+&pka {
+	status = "okay";
+};
+
 &pwr_irq {
 	status = "okay";
 	wakeup-gpios = <&gpiof 8 GPIO_ACTIVE_HIGH>, <&gpioi 3 GPIO_ACTIVE_HIGH>, <0>, <0>, <&gpioi 2 GPIO_PULL_UP>, <0>;
-- 
2.47.2

